sips/sip-015/sip-015-network-upgrade.md

```diff
@@ -0,0 +1,1717 @@
+calculated from a frozen snapshot that precedes this vote, ensure that a whale
+cannot work around this maximum by distributing their STX across many addresses.
+
+## For Miners
+
+There is only one criterion for miners to activate this SIP: they must mine the
+Stacks blockchain up to and past the end of the voting period.  In all reward
+cycles between cycle 45 and the end of the voting period, PoX must activate.
+
+## Examples
+
+### Voting "yes" as a solo Stacker
+
+Suppose Alice has stacked 100,000 STX to `1LP3pniXxjSMqyLmrKHpdmoYfsDvwMMSxJ`
+during at least one of the voting period's reward cycles.  To vote,
+she sends 5500 satoshis for **yes** to `11111111111111X6zHB1ZC2FmtnqJ`.  Then, her 100,000
+STX are tabulated as "yes".
+
+### Voting "no" as a pool Stacker
+
+Suppose Bob has Stacked 1,000 STX in a Stacking pool and wants to vote "no", and
+suppose it remains locked in PoX during at least one reward cycle in the voting
+period.  Suppose his Stacks address is `SP2REA2WBSD3XMVMYS48NJKS3WB22JTQNB101XRRZ`.  To
+vote, he sends 1 uSTX from `SP2REA2WBSD3XMVMYS48NJKS3WB22JTQNB101XRRZ` for **no** to
+`SP00000000000000DSQJTCHE66XE1NHQ`. Then, his 1,000 STX are tabulated as "no."
+
+### Voting "yes" with liquid STX
+
+Suppose Charlie owns 1,000 STX in his wallet at the time of the snapshot,
+and he wants to vote "yes."  To do
+this, he would sign into EcosystemDAO (https://stx.eco) with his Web wallet,
+select the Stacks 2.1 proposal, and select "yes."  If he does not spend or Stack
+any STX over the voting period, then his 1,000 STX are tabulated as "yes."
+
+### Voting "no" with liquid STX that are partially spent
+
+Suppose Danielle owns 1,000 STX in her wallet at the time of the snapshot, and
+she wants to vote "no."  After voting as Charlie had done in the previous
+example, she sends 200 STX to an exchange, and later receives 30 STX, and
+finally unlocks 2,000 STX from a Stacking pool.  In total, only 800 of her STX
+will be tabulated as "no."
+
+### Voting with both liquid STX and Stacked STX
+
+Suppose Erik owns 1,000 STX in his wallet at the time of the snapshot, and he
+wants to vote "yes."  After voting via EcosystemDAO as Charlie and Danielle
+have, he Stacks 900 of his STX in the first voting period reward cycle.  When
+the second reward cycle comes, he votes with his Stacked STX address for "no".
+In this scenario, 1000 of Erik's STX will have counted towards "yes" for the
+non-Stacker vote criteria, and 900 of his STX will have counted towards "no" for
+the Stacker vote criteria.
+
+## Activation Status
+
+This section will be expanded as the voting proceeds, and shall record the
+history of events leading to the activation or rejection of this SIP.
+
+# Reference Implementation 
+
+The reference implementation of this SIP can be found in the `next` branch of
+the Stacks blockchain reference implementation.  It is available at
+https://github.com/stacks-network/stacks-blockchain/tree/next.
+
+# References
```

Comment:
@jcnelson per my other comment, can you add the resources you mentioned in https://github.com/stacksgov/sips/pull/95/files#r1002604198 ?

Reply:
Sure thing